docs/_documentations/remotedeploy-vscode.md

```diff
@@ -10,99 +10,94 @@ parent: installoncloud
 order: 2
 ---
 
-## Connecting VSCode to Remote Codewind
+## Connecting VS Code to Remote Codewind
 
-Locate the Codewind view in VSCode and click the cloud icon to launch the new connection wizard:
+1\. Locate the Codewind view in VS Code and click the cloud icon to launch the new connection wizard:
 
 ![New Connection](./images/remotevs/newConnection.png)
 
-Add a connection name and press enter
+2\. Add a connection name and press `Enter`.
 
 ![Name Connection](./images/remotevs/connectionName.png)
 
-Complete the 3 required fields and save : Gatekeeper URL,  developer username, develper password:
+3\. Complete the three required fields: Gatekeeper URL, developer username, and developer password, and then save.
 
 ![Required Fields](./images/remotevs/connectionCreds.png)
 
-The IDE will then validate the connection and add it to the Codewind panel :
+The IDE validates the connection and adds it to the Codewind panel:
 
 ![Validate settings](./images/remotevs/connectionAdded.png)
 
-At this the IDE and Codewind are connected
+The IDE and Codewind are connected.
 
 ## Adding a deployment registry
 
-Before projects can be deployed on Kubernetes you need to specify a docker registry. In this example we will use DockerHub.  If its not already open navigate to the connection details page via the link :
+Before projects can be deployed on Kubernetes, you need to specify a docker registry. DockerHub is used in this example: 
+
+1\. If it's not already open, navigate to the connection details page by way of the link:
 
 ![Docker Registry](./images/remotevs/connectionSettings.png)
 
-Locate and click 'Open Container Registry Manager':
+2\. Locate and click `Open Container Registry Manager`:
 
 ![Open Registry Manager](./images/remotevs/registryManager.png)
 
-Once the image registry screen opens click '+ Add New'
+3\. In the image registry screen, click `+ Add New`:
 
 ![Adding registry](./images/remotevs/ImageRegistries.png)
 
-Then complete the docker connection details for example if you are using Docker hub enter docker.io and press enter
+4\. Complete the docker connection details. For example. if you are using Docker hub, enter `docker.io` and press `Enter`:
 
 ![New Registry](./images/remotevs/newReg1.png)
 
-Then enter
+Enter:
 
-1. your dockerhub username
-2. your dockerhub password
-3. your repo name which is usually the same as your username
+1. Your dockerhub username.
+2. Your dockerhub password.
+3. Your repo name which is usually the same as your username.
 
-The connection will be tested to validate your credentials and stored in a Kubernetes secret within the Codewind service.
+The connection is tested to validate your credentials and stored in a Kubernetes secret within the Codewind service.
 
 ## Create a new project
 
-Navigate to the Codewind panel and click the + icon beside the new cloud deployment :
+Navigate to the Codewind panel and click the + icon beside the new cloud deployment:
```

Comment:
If this icon has a name, it would be better to refer to it by name instead of as "+". I am assuming a screen reader will catch the name, but it might not catch that the icon looks like a "+".

Reply:
Also for accessibility, remove "beside the new cloud deployment."